src/main/java/com/febfes/fftmback/service/project/ProjectFavoriteServiceImpl.java

```diff
@@ -14,7 +14,7 @@ public class ProjectFavoriteServiceImpl implements ProjectFavoriteService {
 
     @Override
     public void addProjectToFavourite(Long projectId, Long userId) {
-        if (Boolean.FALSE.equals(projectRepository.isProjectFavourite(projectId, userId))) {
+        if (projectRepository.isProjectFavourite(projectId, userId)) {
```

Comment:
```suggestion
        if (!projectRepository.isProjectFavourite(projectId, userId)) {
```

Мы же вроде добавляем в избранное проект, только если он уже не избранный